server2/routes/signupRouter.js

```diff
@@ -15,30 +17,39 @@ router.route('/')
     })
     .post((req, res, next) => {
         const newUser = req.body;
-        //User.sync()
+        // User.sync()
         Promise.all([isDuplicateEmail(newUser), isDuplicateUser(newUser)])
             .then((newUser) => {
                 const saltRounds = 10;
-                const salt = bcrypt.genSaltSync(saltRounds)
-                const hash = bcrypt.hashSync(req.body.password, salt);
-                
-                const test_user = User.create({
+                const salt = bcrypt.genSaltSync(saltRounds);
+                const hash = bcrypt.hashSync(req.body.password, saltRounds);
+
+                const new_user = {
                     id: idGen(),
```

Comment:
Just realised this field is also redundant (it is never used in the verify email since no one is checking the field and it is never saved to database)